lab_4_auto_completion/start.py

```diff
@@ -15,9 +34,116 @@ def main() -> None:
         hp_letters = letters_file.read()
     with open("./assets/ussr_letters.txt", "r", encoding="utf-8") as text_file:
         ussr_letters = text_file.read()
-    result = None
-    assert result, "Result is None"
+    processor = WordProcessor(end_of_sentence_token="<EOS>")
```

Comment:
split lab_work examples and secret solving. You should delete secrets part if you will not solve it untill handover. Remember, that you have to show all aspects of every mark in start.py, load\save too